cli/cmd/plugin/diagnostics/e2e-test/e2e_test.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e_test
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"io"
+	"log"
+	"math/big"
+	"os"
+	"os/exec"
+	"testing"
+)
+
+var myDir string
+var tceRepoPath string
+var tanzuDiagnosticPluginDir string
+var tanzuDiagnosticBin string
+var clusterName string
+var clusterKubeContext string
+var home string
+
+const (
+	colorReset = "\033[0m" // Reset
+	colorBlue  = "\033[34m"
+	colorRed   = "\033[31m" // Fail
+	colorGreen = "\033[32m" // Pass
+	passConst  = "GREEN"
+	failConst  = "RED"
+)
+
+func initializeVariable() {
+	// variables used across functions
+	myDir, _ = os.Getwd()
+	tceRepoPath = os.Getenv("ROOT_DIR")
+	tanzuDiagnosticPluginDir = myDir + "/.."
+	tanzuDiagnosticBin = myDir + "/tanzu-diagnostics-e2e-bin"
+	home = os.Getenv("HOME")
+	rand, _ := rand.Int(rand.Reader, big.NewInt(1000))
+	clusterName = "uc" + rand.String() + "test"
+	clusterKubeContext = "kind-" + clusterName
+}
+
+func TestSetup(t *testing.T) {
+	signal := passConst
+	fmt.Println("-------------------------------", string(colorBlue), "Diagnostic e2e Test", string(colorReset), "---------------------------------------------")
```

Comment:
Personal preference: I'm not a fan of these lines printing out. It goes _abit_ against the paradigm that `go test` has. If the tests has a failure, it should return that error and the `testing` library will handle it. If it succeds, there shouldn't be anything printed to the screen: just a simple `ok` which is also handled by the Go `testing` standard library.

Reply:
Hey @jpmcb I also gave a thought, agree what you mentioned above. The only motive was to make the logs user friendly but as verbosity is not that much and we can have other options too so I am  removing them for now.
Thanks!